tests/vmi_networking_test.go

```diff
@@ -189,29 +189,19 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				virtClient,
 				vmiPod,
 				"compute",
-				[]string{"ip", "address", "show", "k6t-eth0"},
+				[]string{"cat", "/sys/class/net/k6t-eth0/mtu"},
 			)
-			log.Log.Infof("%v", output)
+			log.Log.Infof("k6t-eth0 mtu is %v", output)
 			Expect(err).ToNot(HaveOccurred())
 
-			// The MTU of the Pod network varies, depending on the environment in use. We want to
-			// verify that the actual MTU is from the possible range, which is {1500, 1450}, minus
-			//  50 bytes for vxlan overhead.
-			possibleMtus := []int{1400, 1450}
-			mtuMatch := false
-			expectedMtuString := ""
+			output = strings.TrimSuffix(output, "\n")
+			mtu, err := strconv.Atoi(output)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
I'm not sure that an error of strconv.Atoi is something we should trace with gomega's (the test framework) Expect, and hence fail the test. What is the meaning of Atoi returning an error?
I might be wrong here, but it's worth checking.

Reply:
for example i had to use TrimSuffix, because there was \n and then the atoi failed
we need a valid value for mtu, so it can hint good if there isnt imo, what do you think?

Reply:
OK, makes sense.
So the changes in vmi_networking_test.go LGTM.